### PR TITLE
fix validate ascii

### PR DESCRIPTION
**Pull request by sgaragagghu**

### CODE REVIEW

File: client/utils/utils.c [hunk 1]
```diff
@@ -497,18 +497,19 @@ int validate_ipv4_port(const char *str)
 
 int validate_ascii(const char *str)
 {
-	if(!str || strlen(str) == 0) {
-		return 1;
+	if(str == NULL) {
+		return 1; // why?
```
Comment (xrdavies) [inline client/utils/utils.c]: \0 is valid string.

Comment (sgaragagghu) [reply client/utils/utils.c]: strlen(NULL) will say that there is some issue :D

**Comment by jonano614** [comment]
What is wrong with original function?

**Comment by sgaragagghu** [comment]
bad performance and it doesn't work
`if(c > 126 && c < 32) {`
`if(*str < 32 || *str > 126) {`

**Pull request merged**